docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,175 @@
+ ```
+
+## Deploy the AWS Distro
+
+Deploy the AWS Distro which is a specific distribution of an OTel collector. It converts and dispatches the OTLP-based metrics and traces in the cluster to the AWS-specific format and protocol.
+
+ ```bash
+kubectl -n $K8S_NAMESPACE apply -f aws-otel.yaml
+ ```
+
+## Set Up Kyma Pipelines
+
+Use the Kyma Telemetry module to enable ingestion of the signals from your workloads:
+
+1. Deploy a LogPipeline:
+   ```bash
+   kubectl apply -f - <<EOF
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: LogPipeline
+   metadata:
+     name: aws-cloudwatch
+   spec:
+     output:
+       custom: |
+         Name cloudwatch_logs
+         region \${AWS_REGION}
+         auto_create_group On
+         log_group_template /logs/\$cluster_identifier
+         log_group_name /logs/kyma-cluster         
+         log_stream_template \$kubernetes['namespace_name'].\$kubernetes['pod_name'].\$kubernetes['container_name']
+         log_stream_name from-kyma-cluster
+         log_retention_days 7
+     variables:
+       - name: AWS_ACCESS_KEY_ID
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_ACCESS_KEY_ID
+       - name: AWS_SECRET_ACCESS_KEY
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_SECRET_ACCESS_KEY
+       - name: AWS_REGION
+         valueFrom:
+           secretKeyRef:
+             name: aws-credentials
+             namespace: $K8S_NAMESPACE
+             key: AWS_REGION
+   EOF
+   ```
+   > NOTE: For now, the logs integration is inconsistent with the metrics and traces integration. It will be updated to the common approach as soon as the AWS Distro supports logs and the Kyma Telemetry module uses OTLP logs.
+
+2. Deploy a TracePipeline:
+   ```bash
+   kubectl apply -f - <<EOF
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: TracePipeline
+   metadata:
+     name: aws-xray
+   spec:
+     output:
+       otlp:
+         endpoint:
+           value: http://otel-collector.$K8S_NAMESPACE.svc.cluster.local:4317
+   EOF
+   ```
+
+3. Deploy a MetricPipeline:
+   ```bash
+   kubectl apply -f - <<EOF
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: MetricPipeline
+   metadata:
+     name: aws-cloudwatch
+   spec:
+     input:
+       runtime:
+         enabled: true
+       istio:
+         enabled: true
+       prometheus:
+         enabled: true
+     output:
+       otlp:
+         endpoint:
+           value: http://otel-collector.$K8S_NAMESPACE.svc.cluster.local:4317
+   EOF
+   ```
+
+## Verify the Results
+
+Verify that the logs and metrics are exported to CloudWatch and that the traces are exported to X-Ray.
+
+1. [Install the OpenTelemetry demo application](../opentelemetry-demo/README.md).
+2. In your AWS account, search for **CloudWatch**.
```

Comment:
Again, let's not create click-level documentation for products we don't own.

```suggestion
2. In your AWS account, search for **CloudWatch** and select the same AWS region as the one you have chosen when [creating the secret with AWS credentials](#create-a-secret-with-aws-credentials).
3. Verify that the logs and metrics are exported to CloudWatch, and that the traces are exported to X-Ray.
```
Let's link to AWS documentation if there is anything helpful.

Reply:
Maybe instead of "search for", you could also describe a click-level path to reach CloudWatch or have a direct URL.